### DISCUSSION TITLE
[Feature] Support a Desktop Photo app for Linux (and Windows)

### DISCUSSION THREAD

**Discussion by Mannshoch** [immich-app/immich#14430] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I'm working with Linux and my girlfriend use Windows. I use Shotwell on Linux and manage all Photos on Immich.
As cool my girlfriend finds this Shotwell app. It does not work on Windows. Under Linux I permanently have to deal with mounting the immich library path. (mounting smb or sftp with a notebook is horrible under Linux. Mounts freeze or get lost, or I need even to reboot if I close the lid and left the W-Lan)

My wish is:
Choose an app e.g. Shotwell and create an add-on for it, so I do not need to mount. If you get it work in a way I have not to deal with network mounts, It could maybe a motivation to switch to Linux for my girlfriend.
Second-best option would be, I f you support a photo app for Windows.

Main use in Shotwell:
- Tag Photos general
- Tag Photos for a Photoproject (like photobook, or a photo calendar for an x-mas present)
- Create an event to group photos together. (onhold since I use Immich to store photos)
- Correct creation Time and date e.g. on Whatsapp images.

### Platform

- [ ] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#1427

**Reply by Mannshoch** [reply]
for the record. currently the surface of the web front end is way to slow. (immich runs on a synology nas)
